internal/exec/util/passwd.go

```diff
@@ -218,11 +218,13 @@ func (u Util) AuthorizeSSHKeys(c types.PasswdUser) error {
 		if err != nil {
 			return fmt.Errorf("failed to set SSH key: %v", err)
 		}
-		journal.Send(fmt.Sprintf("wrote ssh authorized keys file for user: %s", c.Name), journal.PriInfo, map[string]string{
+		if err := journal.Send(fmt.Sprintf("wrote ssh authorized keys file for user: %s", c.Name), journal.PriInfo, map[string]string{
 			"IGNITION_USER_NAME": c.Name,
 			"IGNITION_PATH":      path,
 			"MESSAGE_ID":         ignitionSSHAuthorizedkeysMessageID,
-		})
+		}); err != nil {
+			return err
```

Comment:
Likewise, should be wrapped.  Though I'm not sure we should consider it a fatal error if we can't log a journal message.

Reply:
How about this one:
```
		_ = journal.Send(fmt.Sprintf("wrote ssh authorized keys file for user: %s", c.Name), journal.PriInfo, map[string]string{
			"IGNITION_USER_NAME": c.Name,
			"IGNITION_PATH":      path,
			"MESSAGE_ID":         ignitionSSHAuthorizedkeysMessageID,
		})
```